Braintree.xcodeproj/project.pbxproj

```diff
@@ -1597,8 +1602,9 @@
 				3B1C529529D637F400B68A58 /* BTVenmoAnalytics_Tests.swift */,
 				428F48E32624C9B700EC8DB4 /* BTVenmoAppSwitchRedirectURL_Tests.swift */,
 				4245D668256C255F00F1A413 /* BTVenmoAppSwitchReturnURL_Tests.swift */,
-				A71F7DE61B6180A3005DA1B0 /* BTVenmoClient_Tests.swift */,
 				4228D8682624E5C3001D2564 /* BTVenmoRequest_Tests.swift */,
+				09357DCA2A2FBEC10096D449 /* BTVenmoLineItem_Tests.swift */,
+				A71F7DE61B6180A3005DA1B0 /* BTVenmoClient_Tests.swift */,
```

Comment:
we can revert this, these files should be sorted by name

Reply:
We should try [this](https://stackoverflow.com/a/32470246) one day. I wonder how well it works.